### DISCUSSION TITLE
revert back to Default

### DISCUSSION THREAD

**Discussion by ghost** [Abdelrhman-AK/WinPaletter#187] [General]
Hi,
i downloaded your application and i tried some themes and different settings. I want to revert back to my original windows settings, How can i do that? i have removed the themes but the Font size of Icons is still small ! The current tab text on my Browser is also small and it isn't going back to default. please Help. 
thanks

**Comment by ghost** [comment]
![image](https://github.com/Abdelrhman-AK/WinPaletter/assets/135421544/6afe0a43-3440-4e10-9f26-7def28a24daa)
see, this text is still small.

**Comment by Abdelrhman-AK** [comment]
- If you use a high monitor DPI: set it to 100, do the following steps, logoff and logon, reset your previous DPI.

1. Create a new theme dependent on default Windows:
https://github.com/Abdelrhman-AK/WinPaletter/wiki/Main-form-parts#toolbar

2. Open metrics and fonts, enable the toggle, open from default Windows, press on Load into current theme:
https://github.com/Abdelrhman-AK/WinPaletter/wiki/Edit-Windows-Metrics-and-Fonts#4-open-from-default-windows

3. Press on apply in main form

**Reply by ghost** [reply]
still the issue persists. i did this and nothing happened

**Reply by Abdelrhman-AK** [reply]
- Do you use a high screen DPI?
- Do you have an old screenshot, so I can understand the issue completely?

**Reply by Abdelrhman-AK** [reply]
Anyway, you can increase font size in metrics and fonts.
Use `Segoe UI` with size `10` or `11` in affected parts like icons.

**Reply by ghost** [reply]
i'm on laptop and the recommended display size is 150%(i use default). i dont have any screenshot unfortunately.
ok i'm gonna try increasing font size

**Reply by Abdelrhman-AK** [reply]
I faced this issue while testing metrics and fonts last year.

Setting DPI to 100, logingoff and logingon, restoring default Windows theme and setting fonts to defaults, loging off, logging on and resetting to another DPI solved the problem for me.
